crates/fluvio-cluster/build.rs

```diff
@@ -1,28 +1,21 @@
 use std::process::Command;
+use std::path::Path;
 
 fn main() {
-    if let Ok(verpath) = std::fs::canonicalize("../../VERSION") {
-        if verpath.exists() {
-            println!("cargo:rerun-if-changed=../../VERSION");
+    println!("cargo:rerun-if-changed=build.rs");
+    let paths = [
+        "../../k8-util/helm/pkg_sys/fluvio-chart-sys.tgz",
+        "../../k8-util/helm/pkg_app/fluvio-chart-app.tgz",
+    ];
+    for path in paths {
+        // Otherwise when these files are missing, we always rebuild.
```

Comment:
It's possible the build should be erroring when these files don't exist, I'm not sure. But locally for me this and dependent crates were just always rebuilding.

Reply:
@EstebanBorai ^